dali/python/nvidia/dali/ops.py

```diff
@@ -291,6 +291,9 @@ def __init__(self):
     def id(self):
         return self._id
 
+def _instantiate_constant_node(constant):
+    return Constant(value = [constant.value], dtype = constant.dtype)
```

Comment:
```suggestion
    return _Constant(value = [constant.value], dtype = constant.dtype)
```

Reply:
That's intended (it should be ops.Constant, but I don't think a module can refer to itself in Python).

Reply:
Logic changed. Now I use the types.Constant _function_.